Files/Dialogs/RestartDialog.xaml.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Diagnostics;
+using Windows.System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Files.Dialogs
+{
+    public sealed partial class RestartDialog : UserControl
+    {
+        public RestartDialog()
+        {
+            this.InitializeComponent();
+        }
+
+        public void Show()
+        {
+            RestartNotification.Show();
+        }
+
+        public void Dismiss()
+        {
+            RestartNotification.Dismiss();
+        }
+
+        private async void YesButton_Click(object sender, RoutedEventArgs e)
+        {
+            await Launcher.LaunchUriAsync(new Uri("files-uwp://home/page=home"));
```

Comment:
It is important to restore the session so that the user does not lose their flow.

Reply:
Is there a way to navigate to settings via uri?

Reply:
implemented feature